### PR TITLE
Steamworks once again

### PR DESCRIPTION
**Pull request by arthuro555**
See #5401

### CODE REVIEW

**Comment by ClementPasteau** [comment]
@arthuro555 I'll create an experimental-build/ branch with those commits so we can test the code signing.
I'll watch this branch and update the new one with your new commits

**Comment by ClementPasteau** [comment]
PR is here if you want to follow the builds #5440

**Comment by 4ian** [comment]
So seems like people are unresponsive on https://github.com/ceifa/steamworks.js/issues/103
Which means we must investigate and fix this by ourselves. @arthuro555 any chance you can find where in their build process we could add the required flag?

File: .vscode/settings.json [hunk 1]
```diff
@@ -127,5 +127,7 @@
     "flow.useNPMPackagedFlow": true,
 
     // Clang format styling (duplicated in scripts/CMakeClangUtils.txt)
-    "C_Cpp.clang_format_style": "{BasedOnStyle: Google, BinPackParameters: false, BinPackArguments: false}"
+    "C_Cpp.clang_format_style": "{BasedOnStyle: Google, BinPackParameters: false, BinPackArguments: false}",
+    "prettier.prettierPath": "./GDJS/node_modules/prettier",
+    "prettier.configPath": "./GDJS/.prettierrc"
```
Comment (arthuro555) [inline .vscode/settings.json:132]:
```suggestion
    "C_Cpp.clang_format_style": "{BasedOnStyle: Google, BinPackParameters: false, BinPackArguments: false}"
```

**Comment by 4ian** [comment]
Quick update: @arthuro555 if you can provide a github repository we can fork/use for the "official" `@gdevelop/steamworks.js`, that will be helpful in case more changes are needed in the future. 
I can create a GDevelopApp/steamworks.js repo with you having access to it?

**Comment by arthuro555** [comment]
Sure, as I was saying the other day on discord my modifications are on the master branch of my fork of steamworks.js on my github profile: https://github.com/arthuro555/steamworks.js

**Comment by arthuro555** [comment]
With that, I believe we should be getting good to go? Let's merge it in #5440 once again to make sure it really fixed everything, then we'll be able to merge this for real!

@ClementPasteau Could you please update #5440 ?

**Comment by ClementPasteau** [comment]
Thanks @arthuro555 I've pushed the commits to the experimental branch 👍 If it passes we'll be able to merge

**Comment by arthuro555** [comment]
Alright, since the build succeeded, this is now ready to merge!

**Comment by ClementPasteau** [comment]
Closing as merged in #5440

**Pull request closed without merging**